### DISCUSSION TITLE
How can I uninstall multi account containers?

### DISCUSSION THREAD

**Discussion by dodalovic** [mozilla/multi-account-containers#2668] [Q&A]
Hi, I tried to uninstall it, but it's always there, whatever I do. 

Please, advice. 
Thank You.

**Comment by groovecoder** [comment]
Can you describe with more details how you uninstalled it, and then what you see that makes you think it's still installed?

One possibility is: if you un-install Multi-Account Containers, but you still have another Container add-on installed (e.g., Facebook Container), then you will still see the Containers UI in Firefox.

**Reply by mila6452** [reply]
I have the same problem. In FF I have removed the  Multi-Account Container from the "_Extensions_". Not disabled, but removed. The extension icon disappeared from the right top as expected. However the right click pop up list to open a new tab remained. If I select the last one "_Manage containers_", than I found everything there. All the container functions are working. Only the extension icon disappeared. I have no other container extension installed (never before).

**Reply by mcepl** [reply]
Multi-Account Containers are now part of the Firefox itself. Go to about:preferences and close to the top of all preferences there is a check-box switching the functionality off.
